### ISSUE TITLE
[Request] Guests retained per event

### ISSUE THREAD

**Issue by UnsterblichMedic** [derekantrican/GAS-ICS-Sync#468]
### Is your feature request related to a problem? Please describe.

During sync, changed events override gcal event completely including invited guests.

### Describe the solution you'd like

A variable option (true or false) to allow local guest invites on the gcal and not overwrite that during sync.  So if you invite guests to an event on the synced calendar in gcal, as it updates those guests remain instead of being completely removed.

**Comment by jonas0b1011001** [comment]
I can't think of way for us to track if a specific change was made manually, so this is a though request.

1. Maybe using `modifyExistingEvents = false` is viable for your use case?
or
2. You can test if changing
https://github.com/derekantrican/GAS-ICS-Sync/blob/e0f8c14ffc8ebadcba9e12fc9307c789f83367af/Helpers.gs#L615
to
`return Calendar.Events.patch(newEvent, targetCalendarId, calendarEvents[index].id);`
has the intended effect. This will only update Event properties that exist in the source event, so as long as the source event does not have any guest info, it _should_ persist.

**Comment by UnsterblichMedic** [comment]
Perfect!  That worked great.  I changed the return Calendar.Events.update to return Calendar.Events.patch.  Added a guest to the event and change the event on the other end.  It updated as expected and retained the guest. 

I was actually manipulating the next part of that code modifiedEvents.push.  I think I was trying to add to the array something like oldEvent.Guests, oldEvent.Guests to try and retain the old event guests list.  It was not working.

Thank you so much for your assistance.
